### ISSUE TITLE
Cloud SQL does not support multiple deny_maintenance_period block. Update variable description

### ISSUE THREAD

**Issue by Klopfy** [terraform-google-modules/terraform-google-sql-db#546]
### TL;DR

In the module, we can set a list of deny_maintenance_period: https://github.com/terraform-google-modules/terraform-google-sql-db/tree/master/modules/mysql since [#399](https://github.com/terraform-google-modules/terraform-google-sql-db/pull/399).

However, according to the [documentation](https://cloud.google.com/sql/docs/mysql/set-maintenance-window#configure-deny-maintenance), we cannot set multiple deny maintenance period at the same time:
> You can configure a maximum of one deny maintenance period on a Cloud SQL instance at any given time.

I have an according terraform output.

### Expected behavior

Unless I missed something, I would not have expected multiple deny_maintenance_period support in the module (code and doc): https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/mysql/main.tf#L105-L112

### Observed behavior

```
Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Too many deny_maintenance_period blocks
│
│   on .terraform/modules/sql_db_mysql/modules/mysql/main.tf line 107, in resource "google_sql_database_instance" "default":
│  107:       content {
│
│ No more than 1 "deny_maintenance_period" blocks are allowed
``` 

### Terraform Configuration

```hcl
deny_maintenance_period = [
    {
    start_date = "2023-12-10"
    time       = "03:00:00"
    end_date   = "2023-12-12"
    },
    {
      start_date = "2023-12-18"
      time       = "03:00:00"
      end_date   =  "2023-12-20"
    }
  ]
```

### Terraform Version

```sh
$ terraform --version
Terraform v1.6.4
on darwin_amd64
+ provider registry.terraform.io/hashicorp/google v5.7.0
+ provider registry.terraform.io/hashicorp/google-beta v5.7.0
+ provider registry.terraform.io/hashicorp/null v3.2.2
+ provider registry.terraform.io/hashicorp/random v3.5.1
+ provider registry.terraform.io/hashicorp/vault v3.23.0
```

**Comment by imrannayer** [comment]
@Klopfy thx for posting this. Although it accepts one value but it is a block in API. Thats why it was codes as list instead of single object. As of now there is not harm in leaving it as it is but it will be a good idea to update variable description so document will show that list accepts only one value.
